build.rs

```diff
@@ -45,6 +45,13 @@ fn main() {
 
   let gn_target;
 
+  let is_cross_compile = env::var("CARGO_CFG_TARGET_ARCH").unwrap().as_str()
+    != env::var("HOST")
+      .unwrap()
+      .as_str()
+      .split("-")
+      .collect::<Vec<&str>>()[0];
+
```

Comment:
Since we only support two architectures, let's store `CARGO_CFG_TARGET_ARCH` into a variable and add an assert here, so that future readers have some idea of what this value should look like:
```rust
assert!(arch == "x86_64" || arch == "aarch64");
```

Reply:
This code is designed to support any scenario where: host arch != target arch. That would kinda defeat the purpose.
I don't see any reason why this has to be confined to two target architectures. I do think that It could use a comment though.